ingest/src/main/java/com/connexta/ingest/config/ApplicationConfiguration.java

```diff
@@ -43,4 +44,18 @@ public CommonsRequestLoggingFilter requestLoggingFilter() {
   public IngestService ingestService(StoreClient storeClient, TransformClient transformClient) {
     return new IngestServiceImpl(storeClient, transformClient);
   }
+
+  @Bean
+  public StoreClient storeClient(
+      RestTemplate restTemplate, @Value("${endpointUrl.store}") String storeEndpoint) {
```

Comment:
✏️ Add the validators (`@NotNull` and `@NotEmpty`) back to all of the constructor args